src/js/themes/hpe.js

```diff
@@ -714,6 +714,8 @@ const buildTheme = (tokens, flags) => {
       hover: { background: 'background-hover', heading: { color: undefined } },
       border: undefined,
       icons: { collapse: Up, expand: Down, color: 'text' },
+      icon: { container: { pad: 'xsmall' } },
```

Comment:
Since the previous pad only applied to "horizontal", can you update these to be the same?
```suggestion
      icon: { container: { pad: { horizontal: 'xsmall' } } },
```